QuadradiusR/scripts/full_power_list_row.gd

```diff
@@ -0,0 +1,35 @@
+extends HBoxContainer
```

Comment:
this is a single entry in the powerup list, it could be merged with the other entry menu class (piece power row)